tests/import_test.go

```diff
@@ -0,0 +1,35 @@
+package tests_test
+
+import (
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"kubevirt.io/containerized-data-importer/tests/framework"
+	"kubevirt.io/containerized-data-importer/tests/utils"
+)
+
+const (
+	testSuiteName           = "Importer Test Suite"
+	namespacePrefix         = "importer"
+	waitForCDIImportTimeout = 30 * time.Second
+)
+
+var _ = Describe(testSuiteName, func() {
+	f, err := framework.NewFramework(namespacePrefix)
+	if err != nil {
+		Fail("Unable to create framework struct")
+	}
+
+	Describe("Creating standard PVC should not trigger importer", func() {
+		It("PVC should remain empty after being created", func() {
+			pvc, err := f.CreatePVCFromDefinition(utils.NewPVCDefinition("no-import", "1G", nil, nil))
+			Expect(err).ToNot(HaveOccurred())
+			// Wait a while to see if CDI puts anything in the PVC.
+			time.Sleep(waitForCDIImportTimeout)
```

Comment:
@jeffvance I am sure you are not a fan of the sleep here, but for this particular test I see no other option. We are basically verifying that nothing happens. So we need to pick a particular time on how long to wait for 'nothing' to change.

Reply:
understood.